components/cam/bld/namelist_files/use_cases/2010_scream_lr.xml

```diff
@@ -0,0 +1,187 @@
+<mam_mom_mixing_state>3</mam_mom_mixing_state>
+<mam_mom_cycle_yr  >1                                                                    </mam_mom_cycle_yr>
+<mam_mom_datapath  >'atm/cam/chem/trop_mam/marine_BGC/'                                  </mam_mom_datapath>
+<mam_mom_datatype  >'CYCLICAL'										                     </mam_mom_datatype>
+<mam_mom_filename  >'monthly_macromolecules_0.1deg_bilinear_latlon_year01_merge_date.nc' </mam_mom_filename> <!-- Using the 2000 file, for now -->
+<mam_mom_fixed_tod >0											                         </mam_mom_fixed_tod>
+<mam_mom_fixed_ymd >0											                         </mam_mom_fixed_ymd>
+<mam_mom_specifier >'chla:CHL1','mpoly:TRUEPOLYC','mprot:TRUEPROTC','mlip:TRUELIPC'		 </mam_mom_specifier>
+
+<!-- Stratospheric ozone (Linoz) updated using CMIP6 input4MIPS GHG concentrations -->
+<!-- COPIED FROM "CAM" USE CASE; TODO: IS THIS NECESSARY FOR SCREAM? -->
+<chlorine_loading_file      >atm/cam/chem/trop_mozart/ub/Linoz_Chlorine_Loading_CMIP6_0003-2017_c20171114.nc</chlorine_loading_file>
+<chlorine_loading_fixed_ymd >20100101</chlorine_loading_fixed_ymd>
+<chlorine_loading_type      >FIXED</chlorine_loading_type>
+<linoz_data_cycle_yr        >2010</linoz_data_cycle_yr>
+<linoz_data_file            >linoz1850-2015_2010JPL_CMIP6_10deg_58km_c20171109.nc</linoz_data_file>
+<linoz_data_path            >atm/cam/chem/trop_mozart/ub</linoz_data_path>
+<linoz_data_type            >CYCLICAL</linoz_data_type>
+
+<!-- Turn off ozone dry deposition, as Linoz O3v2 and ozone are not separated for now. Need to turn on ozone dry deposition when interactive tropospheric chemistry is implemented -->
+<drydep_list            >'H2O2', 'H2SO4', 'SO2'</drydep_list>
+
+<!-- sim_year used for CLM datasets and SST forcings -->
+<sim_year>2015</sim_year>
+
+<!-- Land datasets set in components/clm/bld/namelist_files/use_cases/1850_CMIP6_control.xml -->
+
+<!-- Prescribed aerosol options -->
+<use_hetfrz_classnuc>.false.</use_hetfrz_classnuc>
+<aerodep_flx_type>'CYCLICAL'</aerodep_flx_type>
+<aerodep_flx_datapath>atm/cam/chem/trop_mam/aero</aerodep_flx_datapath>
+<aerodep_flx_file nlev="72">mam4_0.9x1.2_L72_2000clim_c170323.nc</aerodep_flx_file>
+<aerodep_flx_file nlev="128">mam4_0.9x1.2_L128_2000clim_c191106.nc</aerodep_flx_file>
+<aerodep_flx_cycle_yr>01</aerodep_flx_cycle_yr>
+<prescribed_aero_type>'CYCLICAL'</prescribed_aero_type>
+<prescribed_aero_datapath>atm/cam/chem/trop_mam/aero</prescribed_aero_datapath>
+<prescribed_aero_file nlev="72">mam4_0.9x1.2_L72_2000clim_c170323.nc</prescribed_aero_file>
+<prescribed_aero_file nlev="128">mam4_0.9x1.2_L128_2000clim_c191106.nc</prescribed_aero_file>
+<prescribed_aero_cycle_yr>01</prescribed_aero_cycle_yr>
+
+<!-- Turn on temperature clipping for RRTMGP -->
+<rrtmgp_clip_temperatures> .true. </rrtmgp_clip_temperatures>
+
+<!-- Do radiation every five minutes for ne1024 -->
+<iradsw hgrid="ne1024np4"> 4 </iradsw>
+<iradlw hgrid="ne1024np4"> 4 </iradlw>
```

Comment:
@brhillman - this is for the LR compset, so this won't be rad every 5 min. What is intended?

Reply:
Mostly thinking about the hydrostatic vs non-hydrostatic, or deep convection vs no deep convection tests that have been talked about, and minimizing subtle differences between the compsets that we later forget about and cause us to scratch our heads figuring out what went wrong. I was concerned that if someone tried to run ne1024 with the LR compset to get ZM with the hydrostatic mode (which is really the only difference between the LR and HR compsets), then radiation would silently default back to 1 hour. There's a bigger question here though of how divorced from resolution we should consider a compset definition. Generally since there's nothing to keep folks from using any given compset with any given resolution, I don't really like the idea of assuming in the compset definitions that they will only be used with a certain resolution. I get that a certain amount of this is necessary since the use cases define tuning for different resolutions, but radiation frequency in particular is something that seems like it should be more associated with resolution than with compset, at least once we start "resolving" clouds because the frequency of radiation determines how much a column is able to evolve between radiative heating updates. Tangent aside, I'm happy to remove this. The main point was just to make sure that if we are running ne1024, we want higher rad frequency, and that should happen for any compset not just our FSCREAM-HR family of compsets.

Reply:
I'm fine with that logic. It would be nice if we could specify radiation frequency in minutes instead of steps - a 5 min dt with the ne1024 compset becomes a 2 hr rad dt with the ne30 timestep... That said, I don't think there's any good answer here and sticking with what you have seems best.